egui_demo_lib/src/apps/demo/widget_gallery.rs

```diff
@@ -157,6 +160,19 @@ impl WidgetGallery {
         ui.add(egui::Slider::new(scalar, 0.0..=360.0).suffix("°"));
         ui.end_row();
 
+        ui.add(doc_link_label("ProgressBar", "ProgressBar"));
+        let progress = *scalar / 360.0;
+        let progress_bar = egui::ProgressBar::new(progress)
+            .text(format!("{}%", (progress * 100.0) as usize))
```

Comment:
This seems like something so common that we should have a helper for it, e.g. `.show_percentage()`

Reply:
Added in 1d0d86f